apps/meteor/app/lib/server/functions/validateCustomFields.js

```diff
@@ -44,5 +44,9 @@ export const validateCustomFields = function (fields) {
 				method: 'registerUser',
 			});
 		}
+
+		if (field.searchable && fieldValue !== true && fieldValue !== false) {
```

Comment:
Will only boolean fields be searchable?

Reply:
Btw, not sure 🤔 is this file used by omnichannel?

Reply:
1. Yeah, but fieldValue seems to hold the actual value of the field. So by checking fieldValue !== true & false, aren't you validating that the value of the filed _is_ a boolean, instead of the searchable prop? (which is on "field.searchable"?)
2. Should we modify the user api on this PR? 👀 i thought this was for visitors, and visitors are not users

Reply:
Reverting changes, you're correct.